src/components/Questionnaire/QuestionnaireEditor.tsx

```diff
@@ -1673,9 +2000,9 @@ function QuestionEditor({
             <div className="bg-gray-100 rounded-lg p-1">
               <div className="flex items-center justify-between mb-2">
                 <Label className="text-gray-950 font-semibold">
-                  {question.questions?.length || 0} Sub-Question
-                  {question.questions?.length !== 1 ? "s " : " "}
-                  (for the "{text}" Group)
+                  {question.questions?.length || 0} {t("sub_question")}
+                  {question.questions?.length !== 1 ? t("s") : " "}
+                  {t("for_group", { text })}
```

Comment:
Don't break up words for translations, wouldn't work well for other languages.

Instead keys combining both (singular/plural versions to account for the "s" above).